packages/components/psammead-sitewide-links/src/List/index.jsx

```diff
@@ -15,6 +15,11 @@ import {
 
 import Link from '../Link';
 
+// Gets the number of grid rows, taking into account the
+// first-child in the grid being separate, on its own row.
+const getRowCount = (children, columns) =>
+  Math.ceil((children.length - 1) / columns) + 1;
```

Comment:
```suggestion
  Math.ceil((React.Children.count(children) - 1) / columns) + 1;
```
This handles single or zero components more gracefully than `.length`
https://reactjs.org/docs/react-api.html#reactchildrencount

Reply:
Ohhh good spot, my bad :see_no_evil: